src/test/java/api/requests/scenarios/LoanDueDatesAfterRecallTests.java

```diff
@@ -793,4 +802,46 @@ public void itemRecalledThenCancelledAndNextRecallDoesNotChangeDueDate() {
     assertThat("due date after recall should not change the renewal due date",
         recalledRenewalDueDate, is(renewalDueDate));
   }
+
+  @Test
+  public void shouldNotExtendLoanDueDateIfOverdueLoanIsRecalled() {
+    final IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+
+    final Period loanPeriod = Period.weeks(3);
+    setFallbackPolicies(new LoanPolicyBuilder()
+      .withName("Can Circulate Rolling With Recalls")
+      .withDescription("Can circulate item With Recalls")
+      .rolling(loanPeriod)
+      .unlimitedRenewals()
+      .renewFromSystemDate()
+      .withRecallsMinimumGuaranteedLoanPeriod(Period.weeks(2))
+      .withRecallsRecallReturnInterval(Period.months(2)));
+
+    final DateTime loanCreateDate = DateTime.now(DateTimeZone.UTC)
```

Comment:
Personally, I would statically import the timezone.

```suggestion
    final DateTime loanCreateDate = DateTime.now(UTC)
```